robocup_knowledge/src/robocup_knowledge/environments/robotics_testlabs/common.py

```diff
@@ -52,6 +49,7 @@
     {'category': 'drink',               'name': 'beer',             'color': 'brown',           'volume': 388,      'weight': 100},
     {'category': 'drink',               'name': 'bifrutas',         'color': 'blue',            'volume': 378,      'weight': 100},
     {'category': 'drink',               'name': 'coke',             'color': 'red',             'volume': 376,      'weight': 335},
+    {'category': 'drink',               'name': 'coke_zero',        'color': 'red',             'volume': 376,      'weight': 335},
```

Comment:
Is it in the image recognition?

Reply:
Last time I checked it was.

Reply:
Last time I checked (just now) it was not.
I removed this line